packages/ui/src/components/Card/cardButton.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames'
+import React from 'react'
+
+import './cardCheckbox.less'
+import { CardBody, CardBodyProps } from './_utils/cardBody'
+import { CardHeader, CardHeaderProps } from './_utils/cardHeader'
+import { CARD_CLASS_PREFIX } from './_utils/constants'
+import { Card, CardProps } from './card'
+
+
+export type CardButtonProps = Pick<CardProps, 'accent' | 'disabled'> & {
+    header: CardHeaderProps
```

Comment:
Why can't we set a body without a header?

Reply:
Made header optional